src/neuroconv/datainterfaces/ecephys/requirements.txt

```diff
@@ -1,4 +1,4 @@
-neo @ git+https://github.com/NeuralEnsemble/python-neo@50abfbce832edb1c0f8364ac34c8a6442e1a3edf
-spikeinterface @ git+https://github.com/SpikeInterface/spikeinterface.git@4346eff5706408435e8aa8345be22fdd452afa27
+neo>=0.11
+spikeinterface>=0.95.1
 spikeextractors>=0.9.10
 probeinterface<=0.2.9 # https://github.com/catalystneuro/neuroconv/issues/127
```

Comment:
Can you also try to relax the upper bound on probeinterface? Does latest of that play well with latest of SI now?

Reply:
Actually looks like SI pins to probeinterface >= 0.2.10 anyway, so you can probably remove that altogether here

Reply:
oh yeah, sorry I went ahead and changed it to 0.2.10 or higher. I just finally would like to see whether the tests are passing or not 😄 from the local install that I was testing SI is installed with `neo-0.11.0` and `probeinterface-0.2.10`